spykes/datasets.py

```diff
@@ -29,25 +30,23 @@ def load_reward_data(dpath='~/spykes_data/reward/'):
     sess_four_mat: .mat file
         Monkey M, Session 4
     """
-
+    dpath = os.path.expanduser(dpath)
     if not os.path.exists(dpath):
         os.makedirs(dpath)
 
     fname = os.path.join(dpath, 'Mihili_07112013.mat')
 
     if not os.path.exists(fname):
 
-        urllib.urlretrieve('https://ndownloader.figshare.com/files/5652051',
-                           fname)
+        _fetch_file('https://ndownloader.figshare.com/files/5652051', fname)
```

Comment:
@choldgraf I would prefer to avoid the dependency if possible. The problem is that when MNE makes a new release, then spykes may break. I'd strongly support copying from MNE if possible. Haven't had a chance to look at the the internals of `_fetch_file` yet, but will do.